README.md

```diff
@@ -80,6 +80,7 @@ Rows are listed in a row array as plain objects. Each object contains the row da
     - `row`: *(type: Object)* A vue based object for adding classes.
     - `<column-selector>`: *(type: Object)* A vue based object for adding classes. The column selector works as explained in [styling](#styling),
       without the use of slashes to distinguish columns and rows.
+- `_exactMatch`: *(type: boolean, default: `false`)* Indicates row that exactly matches a filter.
```

Comment:
All the other rows properties, documented here, list configurable row parameters. In contrast `_exactMatch` is modified internally while using the component, so there is no point to set it externally (changes would be overwritten). Therefore, I think, we should not document it.